test/test_pydot.py

```diff
@@ -226,14 +226,14 @@ def _render_and_compare_dot_file(self, fdir: str, fname: str) -> None:
 class TestMyRegressions(RenderedTestCase):
     """Perform regression tests in my_tests dir."""
 
-    @parameterized.expand(functools.partial(_load_test_cases, TESTS_DIR_1))
+    @pytest.mark.parametrize("_,fname,path", _load_test_cases(TESTS_DIR_1))
     def test_regression(self, _: T.Never, fname: str, path: str) -> None:
```

Comment:
The first argument to `parameterized.expand` was there because it governed the naming of the test case, but if that's not necessary/useful with `pytest.mark.parametrize` we can just eliminate it, from both the `_load_test_cases()` returns and the arguments list. That'll save a bunch of special-case code, too.